SECURITY.md

```diff
@@ -1,20 +1,102 @@
 # Security Policy
 
-## Latest Versions
-
-We advise users to run the most recent mainline or stable release of nginx.
+This document provides an overview of security concerns related to nginx
+deployments, focusing on confidentiality, integrity, availability, and the
+implications of configurations and misconfigurations.
 
 ## Reporting a Vulnerability
 
-Please report any vulnerabilities via one of the following methods
-(in order of preference):
+Please report any vulnerabilities via one of the following methods (in order of
+preference):
 
-1. [Report a vulnerability](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability)
-within this repository. We are using the Github workflow that allows us to
-manage vulnerabilities in a private manner and to interact with reporters
-securely.
+1. [Report a vulnerability](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability) within this
+repository. We are using the GitHub workflow that allows us to manage
+vulnerabilities in a private manner and interact with reporters securely.
 
 2. [Report directly to F5](https://www.f5.com/services/support/report-a-vulnerability).
 
-3. Report via email to security-alert@nginx.org.
-This method will be deprecated in the future.
+3. Report via email to security-alert@nginx.org. This method will be deprecated
+in the future.
+
+### Vulnerability Disclosure and Fix Process
+
+The nginx community requests that all suspected vulnerabilities be reported
+privately via the [Reporting a Vulnerability](SECURITY.md#reporting-a-vulnerability) guidelines. 
+If a publicly released vulnerability is reported, we
+may request to handle it according to the private disclosure process. If the
+reporter agrees, we will follow the private disclosure process.
+
+Security fixes will be applied to all supported stable releases, as well as the
+mainline version, as applicable. We recommend using the most recent mainline or
+stable release of nginx. Fixes are created and tested by the core team using a
+GitHub private fork for security. If necessary, the reporter may be invited to
+contribute to the fork and assist with the solution.
+
+nginx is committed to responsibly disclosing information with sufficient detail,
+such as the CVSS score and vector. Privately disclosed vulnerabilities are
+embargoed by default until the fix is released. Communications and fixes remain
+private until made public. As nginx is supported by F5, we generally follow the
+[F5 security vulnerability response policy](https://my.f5.com/manage/s/article/K4602).
+
+### Vulnerability Disclosure and Fix Service Level Objectives
+
+- We will acknowledge all vulnerability reports within 1 to 3 days.
```

Comment:
is there an existing KB that can be linked here to back this statement?

Reply:
No. This service level objective is for nginx oss issues and does not have a KB article.